### DISCUSSION TITLE
Pulsars' theme to follow the system's theme dark/light

### DISCUSSION THREAD

**Discussion by stefaniecg** [pulsar-edit/.github#126] [Pulsar Support] [answered]
How to enable Pulsars' theme to follow the system's theme dark/light?

This was possible in Atom, but cannot find a way to set it in Pulsar. 
Does anyone know how to do this?

Thank you for the help !

**Comment by Daeraxa** [comment]
The functionality shouldn't be any different between the two as Pulsar should still have all the functionality inherited from Atom? What settings/packages/themes were you using in Atom to change the theme?

**Comment by confused-Techie** [comment]
Yeah I don't think this is a built in feature, it may have been something offered by an additional package you had installed. But please someone correct me if that's incorrect

**Comment by stefaniecg** [comment]
HI guys! Thank you for your replies. 

The problem is i cannot open Atom any more to see my package configuration. 
For some reason it says that it is a malware, then it crashes immediatly.
Don't know why? This happened sudently, see picture attached. 

Does anyone know how can i access my Atom configuration regarding packages?
Where is it stored?

Thank you for the help 

![Screen shot Atom crash](https://user-images.githubusercontent.com/64597312/216810279-cac5b976-1670-4fcc-98c7-ab080a037cb0.png)

**Reply by Spiker985** [reply]
https://github.blog/2023-01-30-action-needed-for-github-desktop-and-atom-users/

**Comment by stefaniecg** [comment, marked as answer]
thank you for the infos @Spiker985 !
So went back to the previous version of Atom, and now i can lauch it.
Now i can see the package i used to follow the system's switch between light and dark is: **[auto-dark-mode 0.6.0](https://web.pulsar-edit.dev/packages/auto-dark-mode)**

Thank you for your help ! ;)

**Reply by confused-Techie** [reply]
Glad you were able to find the package that was doing this, that's awesome!
